Ginger/GingerCore/ALM/RQMCore.cs

```diff
@@ -185,8 +186,11 @@ public override ObservableList<ApplicationPlatform> ApplicationPlatforms
                     //Extract end return ServerURL value from RQM/GeneralData/ServerURL node
                     XmlNode ServerURLNode = RQMSettingsXML.SelectSingleNode("RQM/GeneralData/ServerURL");
                     string serverURL = ServerURLNode.InnerText;
+                    XmlNode IsTestSuiteNode = RQMSettingsXML.SelectSingleNode("RQM/GeneralData/IsTestSuite");
+                    string IsTestSuite = IsTestSuiteNode.InnerText;
                     Dictionary<String, Object> dictionary = new Dictionary<string, object>();
                     dictionary.Add("ServerURL", serverURL);
+                    dictionary.Add("IsTestSuite", IsTestSuite);
                     return dictionary;
```

Comment:
Add check if dictionary already exists, if yes return the same else to continue populate it at the start of function to